src/py/flwr/cli/run/run.py

```diff
@@ -169,10 +169,8 @@ def on_channel_state_change(channel_connectivity: str) -> None:
 
 
 def _run_without_superexec(
-    config: Dict[str, Any], federation: Dict[str, Any], federation_name: str
+    directory: Optional[Path], federation: Dict[str, Any], federation_name: str
```

Comment:
Let's name it `app_path` since the expected object is of type `Path` (we should perhaps rename the `flwr run` `directory` arg as well at some point):
```suggestion
    app_path: Optional[Path], federation: Dict[str, Any], federation_name: str
```